cfme/tests/cloud/test_tenant.py

```diff
@@ -34,14 +40,23 @@ def test_tenant_crud(tenant):
     Metadata:
         test_flag: tenant
     """
+
     tenant.create(cancel=True)
     assert not tenant.exists()
 
     tenant.create()
     assert tenant.exists()
 
+    tenant.update({'name': 'my_new_name'})
+    tenant.wait_for()
```

Comment:
Why is there a need for a wait_for here?? Is this an openstack tenant? Are renaming it inside openstack? If so, the wait_for function probably doesn't need to be there and instead you can just put a wait around .exists in the update method.